Source/Scene/Cesium3DTile.js

```diff
@@ -276,8 +259,40 @@ define([
     /**
      * DOC_TBA
      */
-    Cesium3DTile.prototype.isRefinable = function() {
-        return this._numberOfUnrefinableChildren === 0;
+    Cesium3DTile.prototype.isRefinable = function(cullingVolume) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(cullingVolume)) {
+            throw new DeveloperError('cullingVolume must be defined');
+        }
+        //>>includeEnd('debug');
+
+        var visibleChildren = [];
+        var child;
+        var k;
+
+        for (k = 0; k < this.children.length; ++k) {
+            child = this.children[k];
+            if (child.visibility(cullingVolume) !== CullingVolume.MASK_OUTSIDE) {
+                visibleChildren.push(child);
+            }
+        }
```

Comment:
This could be made into a prototype function called `getVisibleChildren`. 

Instead of creating a new array every time, maybe store the array as part of `Cesium3DTile`, or even as a scratch array which may be safe now but maybe not later.